list/list.js

```diff
@@ -857,8 +858,8 @@ assign(DefineList.prototype, {
 	forEach: function(cb, thisarg) {
 		var item;
 		for (var i = 0, len = this.length; i < len; i++) {
-			item = this.get(i);
-			if (cb.call(thisarg || item, item, i, this) === false) {
+			item = canReflect.getKeyValue(this, i);
```

Comment:
I think `.get()` should still be used.  It's ok to call methods on ourselves that we know must exist (faster this way too).